folium/features.py

```diff
@@ -413,22 +413,23 @@ class GeoJson(Layer):
                 }
             });
         };
-        var {{ this.get_name() }} = L.geoJson(null, {
-            {%- if this.smooth_factor is not none  %}
-                smoothFactor: {{ this.smooth_factor|tojson }},
-            {%- endif %}
-                onEachFeature: {{ this.get_name() }}_onEachFeature,
-            {% if this.style %}
-                style: {{ this.get_name() }}_styler,
-            {%- endif %}
-        }).addTo({{ this._parent.get_name() }});
+        var options = {
+        {%- if this.smooth_factor is not none  %}
```

Comment:
Why do you remove the indentation here?

Reply:
May be automatic linting, I didn't see it. Sorry